app/src/pages/[resourceType]/[id].tsx

```diff
@@ -84,30 +83,45 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
     }
   }, [dataRequirements]);
 
-  const draftMutation = trpc.draft.createDraft.useMutation({
+  const successNotification = (data: { draftId: string }, createdFromArtifact: boolean) => {
+    const message = createdFromArtifact
+      ? `Draft of ${jsonData.resourceType}/${jsonData.id} successfully created`
+      : `${resourceType} successfully created`;
+    notifications.show({
+      title: `${resourceType} Created!`,
+      message: message,
+      icon: <CircleCheck />,
+      color: 'green'
+    });
+    router.push(`/authoring/${resourceType}/${data.draftId}`);
+    ctx.draft.getDraftCounts.invalidate();
+  };
```

Comment:
I noticed that the release notification gets called on the measure and its child artifacts upon release, which is pretty cool because that gives another indication that the child artifacts are being accounted for. When I drafted a measure, I believe I only got a notification of the measure being drafted, and not of the child library also being drafted. What are your thoughts in regard to adding a notification for when a child is drafted? Do you think that would take a lot of re-working and be out out scope? At first glance it seems like this might be more difficult to achieve as opposed to the release notifications because the draft notification is built into the `[id]` page and not a component.

Reply:
Thank you for commenting about this! Unfortunately, it was quite a bit of reworking but it was all very necessary changes that not only accomplish what you suggested here, but also make the code a lot cleaner.